code/obj/vehicle.dm

```diff
@@ -10,8 +10,33 @@ Contains:
 -Forklift
 */
 
+//////////////////////////////// Vehicle Defines ///////////////////////////////////////
+#define MINIMUM_EFFECTIVE_DELAY 0.001 //do not set to zero or shit breaks
+
+#define V_DO_MOVE(src, user, dir) do { \
```

Comment:
documentation please

how is this different from theforklift thing

Reply:
The forklift was a special case because we have to handle the rider's overlay in kind of a weird way that you don't really have to for other vehicles, so I had to trim some of the code from the normal movement handler in order to not cause weird graphical glitches.

I *could* have put extra checks into V_DO_MOVE to handle this, but no other vehicle does this and I didn't want to make the macro less performant for this one edge case, especially since it's called pretty frequently.

Uhhh, what would you be looking for in terms of documentation? I thought I already named all the variables and such in a pretty descriptive way and included comments where necessary, so I'm not 100% sure what's missing.

Reply:
this define isn't documented at all? what does it do?

Reply:
Alright, I'll write up some documentation later today.

Reply:
done